src/smart-components/survey-editing/survey-editor-s.js

```diff
@@ -398,7 +398,6 @@ const SurveyEditor = ({ closeUrl, search, portfolioItem }) => {
             disableDrag
             disableAdd
             schemaTemplate={baseSchema}
-            mode="subset"
```

Comment:
What does this do? Curious how this change relates to approval

Reply:
I'll remove this change.